src/msg/async/Stack.h

```diff
@@ -211,6 +211,8 @@ enum {
   l_msgr_recv_encrypted_bytes,
   l_msgr_send_encrypted_bytes,
 
+  l_msgr_all_read_lat,
```

Comment:
IIUC this is a sum of recv latency from connections. Could you throw some light on use cases / examples of use when such wide aggregate turned out to be helpful?

Reply:
we can use to monitoring:
```
rate(ceph_msgr_all_read_latency_sum{cluster="$cluster"}[5m]) / rate(ceph_msgr_all_read_latency_count{cluster="$cluster"}[5m]
```